cupy/linalg/decomposition.py

```diff
@@ -224,64 +224,83 @@ def svd(a, full_matrices=True, compute_uv=True):
     # Remark 2: gesvd only supports jobu = 'A' and jobvt = 'A'
     # Remark 3: gesvd returns matrix U and V^H
     # Remark 4: Remark 2 is removed since cuda 8.0 (new!)
-    n, m = a.shape
-
-    # `a` must be copied because xgesvd destroys the matrix
-    if m >= n:
-        x = a.astype(a_dtype, order='C', copy=True)
-        trans_flag = False
-    else:
-        m, n = a.shape
-        x = a.transpose().astype(a_dtype, order='C', copy=True)
+    
+    n, p = a.shape
```

Comment:
Ye soz it was a quick refactor to test if my changes actually made SVD faster. So to make my life easier I decided to use n, p (statistics convention) rather than the general m, n.

Reply:
Could you change it back to the previous naming convention?